### PR TITLE
add easyblock for WRF-Fire

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

File: easybuild/easyblocks/w/wrf_fire.py [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+        self.build_in_installdir = True
+
+    def extract_step(self):
+        """Extract WRF-Fire sources."""
+        self.cfg.update('unpack_options', '--strip-components=1')
+        super(EB_WRF_minus_Fire, self).extract_step()
+
+    def configure_step(self):
+        """Custom configuration procedure for WRF-Fire."""
+
```
Comment (wpoely86) [inline easybuild/easyblocks/w/wrf_fire.py]: less whitespace?

File: easybuild/easyblocks/w/wrf_fire.py [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+        change_dir('WRFV3')
+
+        # define $NETCDF* for netCDF dependency
+        env.setvar('NETCDF', get_software_root('netCDF-Fortran'))
```
Comment (wpoely86) [inline easybuild/easyblocks/w/wrf_fire.py]: no check if it's loaded?

File: easybuild/easyblocks/w/wrf_fire.py [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+        if comp_fam in known_build_type_options:
+            build_type_option = known_build_type_options[comp_fam]
+        else:
+            raise EasyBuildError("Don't know how to figure out build type to select for compiler family %s", comp_fam)
+
+        build_type_question = "\s*(?P<nr>[0-9]+).\s*%s\s*\(%s\)" % (build_type_option, self.cfg['buildtype'])
+        qa = {
+            "Compile for nesting? (1=basic, 2=preset moves, 3=vortex following) [default 1]:": '1',
+        }
+        no_qa = [
```
Comment (wpoely86) [inline easybuild/easyblocks/w/wrf_fire.py]: singe line?

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**